docs/snippets/react/your-component-with-decorator.ts.mdx

```diff
@@ -1,5 +1,5 @@
 ```ts
-// your-component.stories.ts
+// your-component.stories.tsx
```

Comment:
```suggestion
// YourComponent.stories.tsx
```

I notice we use YourComponent and MyComponent in these snippets. Should we standardize on one pattern?

Reply:
Fixed with the latest commit. @domyen . That's something that i need to take care of, I'm going to do a search on the entire documentation and see if we can consolidate some of the snippets and manage them better.